Source/BugsnagStacktrace.h

```diff
@@ -0,0 +1,25 @@
+//
+//  BugsnagStacktrace.h
+//  Bugsnag
+//
+//  Created by Jamie Lynch on 06/04/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class BugsnagStackframe;
+
+/**
+ * Representation of a stacktrace in a bugsnag error report
+ */
+@interface BugsnagStacktrace : NSObject
+
+- (instancetype)initWithTrace:(NSArray<NSDictionary *> *)trace
+                 binaryImages:(NSArray<NSDictionary *> *)binaryImages;
+
+- (NSArray *)toArray;
+
+@property NSMutableArray<BugsnagStackframe *> *trace;
```

Comment:
I think this can be private. It's only used in the class and tests.

Reply:
Moved to a class extension inside `BugsnagStacktrace.m`